启动forge optifine共存版本, 在游戏界面内点击"打开光影包文件夹"时提示找不到文件夹

### 检查项

- [X] 我已在 [Issues 页面](https://github.com/Hex-Dragon/PCL2/issues?q=is%3Aissue+) 和 [常见&难检反馈及问题列表](https://github.com/Hex-Dragon/PCL2/discussions/1930) 中搜索，确认了这一 Bug 未被提交过。

### 描述

使用PCL2 启动 Minecraft 时在菜单的视频子菜单内点击"打开光影包"文件夹，提示 "找不到文件"

PCL2版本: Snapshot _2.7.3_, 能够在 Release _2.7.1.1_ 中复现
Minecraft版本: _1.12.2_
Forge版本: _14.23.5.2847_
Optifine版本: _G5_
Java版本: _1.8.0_362_
系统: Windows 10 ProWorkstation, _22H2_
![image](https://github.com/Hex-Dragon/PCL2/assets/107820172/70e979e7-de8c-4656-a872-5554318443dd)


### 重现步骤

1. 在启动游戏后由主菜单进入选项->视频设置->光影->光影包文件夹
2. 提示 Windows找不到文件'\C:\Programs\PCL2\.minecraft\versions\1.12.2-Forge_14.23.5.2847-Optifine_G5 光影包测试\shaderpacks\'。请确定文件名是否正确后，再试一次。

### 日志与附件

[latest.log](https://github.com/Hex-Dragon/PCL2/files/15376265/latest.log)
[Log1.txt](https://github.com/Hex-Dragon/PCL2/files/15376267/Log1.txt)

> ？

使用HMCL启动该版本无法启动
[摘自HMCL启动的latest.log]

[11:44:31] [main/ERROR]: Unable to launch
java.lang.reflect.InvocationTargetException: null
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_411]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_411]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_411]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_411]
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135) [launchwrapper-1.12.jar:?]
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.12.jar:?]
Caused by: java.lang.NoSuchMethodError: javax.vecmath.Matrix4f: method <init>()V not found
	at net.minecraftforge.client.ForgeHooksClient.<clinit>(ForgeHooksClient.java:420) ~[ForgeHooksClient.class:?]
	at net.minecraft.client.Minecraft.<init>(Minecraft.java:335) ~[bib.class:?]
	at net.minecraft.client.main.Main.main(SourceFile:123) ~[Main.class:?]
	... 6 more
[11:44:31] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:-1]: net.minecraftforge.fml.relauncher.FMLSecurityManager$ExitTrappedException
[11:44:31] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at net.minecraftforge.fml.relauncher.FMLSecurityManager.checkPermission(FMLSecurityManager.java:49)
[11:44:31] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at java.lang.SecurityManager.checkExit(Unknown Source)
[11:44:31] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at java.lang.Runtime.exit(Unknown Source)
[11:44:31] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at java.lang.System.exit(Unknown Source)
[11:44:31] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at net.minecraft.launchwrapper.Launch.launch(Launch.java:138)
[11:44:31] [main/INFO]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

--- a/Plain Craft Launcher 2/Pages/PageSetup/ModSetup.vb
+++ b/Plain Craft Launcher 2/Pages/PageSetup/ModSetup.vb
@@ -86,7 +86,7 @@
         {"LaunchArgumentWindowHeight", New SetupEntry(480)},
         {"LaunchArgumentWindowType", New SetupEntry(1)},
         {"LaunchArgumentRam", New SetupEntry(False, Source:=SetupSource.Registry)},
-        {"LaunchAdvanceJvm", New SetupEntry("-XX:+UseG1GC -XX:-UseAdaptiveSizePolicy -XX:-OmitStackTraceInFastThrow -Dfml.ignoreInvalidMinecraftCertificates=True -Dfml.ignorePatchDiscrepancies=True -Dlog4j2.formatMsgNoLookups=true")},
+        {"LaunchAdvanceJvm", New SetupEntry("-XX:+UseG1GC -XX:-UseAdaptiveSizePolicy -XX:-OmitStackTraceInFastThrow -Djdk.lang.Process.allowAmbiguousCommands=true -Dfml.ignoreInvalidMinecraftCertificates=True -Dfml.ignorePatchDiscrepancies=True -Dlog4j2.formatMsgNoLookups=true")},
         {"LaunchAdvanceGame", New SetupEntry("")},
         {"LaunchAdvanceRun", New SetupEntry("")},
         {"LaunchAdvanceRunWait", New SetupEntry(True)},

